mlflow/utils/databricks_utils.py

```diff
@@ -162,6 +163,20 @@ def get_databricks_host_creds(profile=None):
         config = provider.ProfileConfigProvider(profile).get_config()
     else:
         config = provider.get_config()
+    # if a path is specified, that implies a Databricks tracking URI of the form:
+    # databricks://profile-name/path-specifier
+    if (not config or not config.host) and path:
```

Comment:
what if path is ""? should we support that?

Reply:
Yeah, good point! We currently don't support that, due to the path truthiness stuff. 

I think the problem with supporting it is that an incorrect profile passed in would throw an error related to the lack of a path. So, for example, if you don't have a configured profile `databricks://profile1/`, we currently throw and say that you misconfigured your URI and you should change it. If we supported "", we would attempt to fetch secrets from the Secrets service and throw on the lack of that scope being defined or on the lack of presence of a "host" or "token" key and that case would be difficult to distinguish from scenarios where people weren't even trying to use this feature or multi-workspace registry at all (for example, just in tracking)

Reply:
ok so we just need to document that the secret key prefix cannot be empty